docs/guides/hello.md

```diff
@@ -908,8 +909,8 @@ class GreeterClient {
   </div>
   <div id="python_connect">
 <p>The generated Python code has a helper function that creates a stub given a channel.</p>
-<pre>channel = implementations.insecure_channel('localhost', 50051)
-stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+<pre>channel = grpc.insecure_channel('localhost', 50051)
```

Comment:
`grpc.insecure_channel` takes a single `target` parameter. In this case you want to pass [`'localhost:50051'`](https://github.com/grpc/grpc/blob/46585e23f14b0463c4b7a0d04d72d71228c/examples/python/helloworld/greeter_client.py#L40).